src/devices/cpu/m6800/6800ops.hxx

```diff
@@ -22,6 +22,20 @@ OP_HANDLER( illegal )
 	logerror("m6800: illegal opcode: address %04X, op %02X\n",PC-1,(int) M_RDOP_ARG(PC-1)&0xFF);
 }
 
+//OP_HANDLER( illega2 )
+OP_HANDLER( illega2 )
+{
+	logerror("m6800: illegal 2 bytes opcode: address %04X, op %02X\n",PC-1,(int) M_RDOP_ARG(PC-1)&0xFF);
```

Comment:
I know it's trivial, but could you fix the grammar in the messages?  It should be "2-byte opcode" etc. rather than "2 bytes opcode" etc.  Also, could you please update the message in the illegal 1-byte opcode handler to make the length explicit?